File: lingting-polaris-grpc/src/main/java/live/lingting/polaris/grpc/server/DelayRegister.java

package live.lingting.polaris.grpc.server;

/**
 * @author <a href="mailto:liaochuntao@live.com">liaochuntao</a>
 */
public interface DelayRegister {

	/**
	 * 允许注册
	 * @return 是否可以放通注册动作
	 */
	boolean allowRegis();

}

File: lingting-polaris-grpc/src/main/kotlin/live/lingting/polaris/grpc/server/DelayRegister.kt
package live.lingting.polaris.grpc.server

/**
 * @author [liaochuntao](mailto:liaochuntao@live.com)
 */
interface DelayRegister {
    /**
     * 允许注册
     * @return 是否可以放通注册动作
     */
    fun allowRegis(): Boolean
}
